graderx/graders/courses_config.json

```diff
@@ -17,5 +17,40 @@
       "lab2",
       "lab3"
     ]
+  },
+  "new_course": {
+    "course_type": "stdout",
+    "language": "C++",
+    "labs": [
+      {
+        "name": "Lab1",
+        "runtime": "4",
+        "disable_internet": true,
+        "test_cases": [
+          {
+            "tc_id": "1",
+            "input": "tc1_in_str",
+            "output": "tc1_out_str"
+          },
+          {
+            "tc_id": "2",
+            "input": "tc2_in_str",
+            "output": "tc2_out_str"
+          }
+        ]
```

Comment:
courses_config should remain as it was in the repo because now the frontend won't get cc451 and test_course
also no need to push any courses, so please empty the courses directory to avoid problems during development